### DISCUSSION TITLE
Question regarding termux-services

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#18288] [Q&A] [answered]
Does termux-services run independent of termux, much in the way as termux-job-scheduler does? 
Will it go to 'sleep' or be purged by Android if termux-wake-lock is not enforced or termux isn't 'active'?

**Comment by sylirre** [comment, marked as answer]
`termux-services` configures a Runit service management daemon. Basically it is a service to manage other services.

Yes, it will go to sleep like everything else running in Termux environment when the wake lock was not enabled.
